src/defaultPhrases.js

```diff
@@ -25,13 +25,11 @@ const returnFocusToInput = 'Return to the date input field.';
 const keyboardNavigationInstructions = `Press the down arrow key to interact with the calendar and
   select a date. Press the question mark key to get the keyboard shortcuts for changing dates.`;
 
-// eslint-disable-next-line camelcase
 const chooseAvailableStartDate = ({ date }) => `Choose ${date} as your check-in date. It's available.`;
-
-// eslint-disable-next-line camelcase
```

Comment:
I removed these disable comments as these lines aren't failing that rule anymore

Reply:
Thanks!